README.md

```diff
@@ -1,13 +1,13 @@
-# csg.js
+## csg.js
 
+## Constructive Solid Geometry (CSG) Library
 
 [![GitHub version](https://badge.fury.io/gh/jscad%2Fcsg.js.svg)](https://badge.fury.io/gh/jscad%2Fcsg.js)
 [![Build Status](https://travis-ci.org/jscad/csg.js.svg)](https://travis-ci.org/jscad/csg.js)
 
-> Solid modeling library (2d & 3d)
```

Comment:
I saw that in the "standard" but couldn't determine where to put the one line. Feel free to change.